src/dune_rules/stanzas/parameter.ml

```diff
@@ -27,7 +27,7 @@ let to_library t =
   ; library_flags = Ordered_set_lang.Unexpanded.standard
   ; c_library_flags = Ordered_set_lang.Unexpanded.standard
   ; virtual_deps = []
-  ; wrapped = This (Simple false)
+  ; wrapped = This (Simple true)
```

Comment:
Should this change be in #1963 since it impacts the `library_parameter` stanza opam installation?